src/utils/dates.js

```diff
@@ -10,3 +10,27 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(startDate, endDate) {
```

Comment:
consider keeping your functions atomic and doing "business logic" in firebase.js to compose them

Here's one thought on how to split up the functions: getFutureDate / getDaysBetweenDates / isMoreThanADayOld